engines/ags/lib/freetype-2.1.3/autohint/ahangles.cpp

```diff
@@ -0,0 +1,155 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/***************************************************************************/
+/*                                                                         */
+/*  ahangles.c                                                             */
+/*                                                                         */
+/*    A routine used to compute vector angles with limited accuracy        */
+/*    and very high speed (body).                                          */
+/*                                                                         */
+/*  Copyright 2000-2001, 2002 Catharon Productions Inc.                    */
+/*  Author: David Turner                                                   */
+/*                                                                         */
+/*  This file is part of the Catharon Typography Project and shall only    */
+/*  be used, modified, and distributed under the terms of the Catharon     */
```

Comment:
This is a fun coincidence:

```
[Freetype Project License](https://directory.fsf.org/wiki/License:FreeType) ([#freetype](https://www.gnu.org/licenses/license-list.en.html#freetype))
This is a free software license, and compatible with GPLv3. It has some attribution requirements which make it incompatible with GPLv2.
```

Reply:
The FreeType website says:

FreeType comes with two licenses from which you can choose the one which fits your needs best.

The [FreeType License](https://gitlab.freedesktop.org/freetype/freetype/-/blob/master/docs/FTL.TXT) (FTL) is the most commonly used one. It is a BSD-style license with a credit clause and thus compatible with the GNU Public License (GPL) version 3, but not with the GPL version 2.

The [GNU General Public License](https://gitlab.freedesktop.org/freetype/freetype/-/blob/master/docs/GPLv2.TXT) (GPL), version 2. Use it for all projects which use the GPLv2 also, or which need a license compatible to the GPLv2.